### PR TITLE
chore: type countryCodes object

### PR DESCRIPTION
**Pull request by ansmonjol**
Better with types!

### CODE REVIEW

File: src/countryCodes.ts [hunk 1]
```diff
@@ -0,0 +1,257 @@
+import { CountryCode } from './generated/graphql'
```
Comment (MorganeLecurieux) [inline src/countryCodes.ts:1]: Not a big fan to put that at the root
Why did you remove it from public ?
Alternative could be in `core`

File: src/components/settings/EditOrganizationInformationsDialog.tsx [hunk 2]
```diff
@@ -14,20 +14,19 @@ import {
...
 
-const countryData: { value: string; label: string }[] = Object.keys(CountryCodes).map(
-  (countryKey) => {
-    return {
-      value: countryKey,
-      // @ts-ignore
-      label: CountryCodes[countryKey],
-    }
+const countryData: { value: string; label: string }[] = (
+  Object.keys(CountryCodes) as Array<keyof typeof CountryCodes>
```
Comment (MorganeLecurieux) [inline src/components/settings/EditOrganizationInformationsDialog.tsx:23]: This is used several times, maybe we can export it form the countryCode file
Also, the the type is just an array of strings : `Object.keys(CountryCodes) as string[]`

File: src/components/settings/EditOrganizationInformationsDialog.tsx [hunk 3]
```diff
@@ -14,20 +14,19 @@ import {
...
 const FILE_MAX_SIZE = 800000
 
-const countryData: { value: string; label: string }[] = Object.keys(CountryCodes).map(
-  (countryKey) => {
-    return {
-      value: countryKey,
-      // @ts-ignore
-      label: CountryCodes[countryKey],
-    }
+const countryData: { value: string; label: string }[] = (
```
Comment (MorganeLecurieux) [inline src/components/settings/EditOrganizationInformationsDialog.tsx:22]: This should be the `BasicComboBoxData` type

Comment (ansmonjol) [reply src/components/settings/EditOrganizationInformationsDialog.tsx:22]: Indeed will look into it

File: src/countryCodes.ts (hunk 1 above)
Comment (ansmonjol) [reply src/countryCodes.ts]: Core sound like a good alternative,

updated!

File: src/components/settings/EditOrganizationInformationsDialog.tsx (hunk 2 above)
Comment (ansmonjol) [reply src/components/settings/EditOrganizationInformationsDialog.tsx]: Done! thanks for pointing this

**Review by MorganeLecurieux — APPROVED** [review]
